cfme/tests/webui/test_expression_editor.py

```diff
@@ -0,0 +1,44 @@
+import fauxfactory
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+
+class TestDeleteExpressions(object):
+    filter_value = 'beautifulpotato'
+    filt = "Cloud Volume : Name"
+    collection = "volumes"  # any collection with advanced search
+    destination = "All"
+
+    @pytest.mark.meta(automates=[BZ(1720216)])
+    @pytest.mark.parametrize("operator", ["not", "and", "or"])
+    # remove NOT is 5.11 feature, see BZ(1720216)
+    @pytest.mark.uncollectif(lambda operator, appliance:
+                             operator == "not" and appliance.version < "5.11")
+    def test_delete_operator(self, appliance, operator):
```

Comment:
I'd give this test longer name clearly explaining where it is used.

Reply:
I don't know how to change the test name. Now it's something like  test_expression_editor::test_delete_operator[not]. Test is just about expression editor, which is specified by module name, destination like volumes/all isn't essential.. If you have an idea for clearer name, I can use it of course :)